eox_nelp/user_profile/api/v1/views.py

```diff
@@ -56,14 +57,17 @@ def update_user_data(request):
             # This extra code block is required since the method update_account_settings just
             # allows to update fields defined in the AccountUserSerializer and the AccountLegacyProfileSerializer
             # so some fields like first_name and last_name are not editable in the standad implementation.
-            extra_account_user_fields = getattr(settings, "EXTRA_ACCOUNT_USER_FIELDS", [])
+            # so some fields related ExtraInfo are not editable in the standad implementation.
```

Comment:
Could you fix the typo in the previous line and edit this line since in the context doesn't make sense